otter/test/convergence/test_steps.py

```diff
@@ -43,19 +44,32 @@ def test_create_server(self):
         self.assertEqual(eff.intent, Func(generate_server_name))
         eff = resolve_effect(eff, 'random-name')
         self.assertEqual(
-            eff,
+            eff.intent,
             service_request(
                 ServiceType.CLOUD_SERVERS,
                 'POST',
                 'servers',
                 data={'server': {'name': 'myserver-random-name',
                                  'flavorRef': '1'}},
-                success_pred=has_code(202)))
+                success_pred=has_code(202)).intent)
+
+        self.assertEqual(
```

Comment:
Non-blocking - wondering if failure should be a different test?  I realize it's pretty short - changing the docstring to be more clear about exactly what this test does, and maybe providing a message to `assertEqual` would probably be sufficient too.